### PR TITLE
Add category term mapping and more descriptive text for terms

### PR DESCRIPTION
**Pull request by mickeyryan42**
https://issues.newrelic.com/browse/NR-85162

This PR adds extra context for users in the `CONTRIBUTING.md` doc to advise contributors to include category terms, and links to a query mapping category term display names to values.

Other doc updates will be made in marketplace repo describing the changes for quickstarts in add data

### CODE REVIEW

**Review by d3caf — DISMISSED** [review]
LGTM 👍 It's interesting that we spell it `catalogue` in all these docs but it's `catalog` throughout the codebase 😆

**Comment by mickeyryan42** [comment]
> LGTM 👍 It's interesting that we spell it `catalogue` in all these docs but it's `catalog` throughout the codebase 😆

this was not lost on me either :D

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -615,7 +615,8 @@ keywords:
   - test
   - keyword2
 
-# Terms related to the categories for the data source
+# Terms related to the categories for the data source.
+# NOTE: Although not required, highly recommended because they determine category placement. Otherwise will be in the 'Other' category.  See below table for category term mappings.
```
[the remark this answers is not in the document]
Comment (mickeyryan42) [reply CONTRIBUTING.md]: I agree, updated!

**Comment by brammerl** [comment]
I think we should add some documentation to the quickstart field about how keywords can be used and mapped to certain category terms. 

The quickstarts themselves have category terms that we correlate to the keywords that a user submits. So if the user wants a certain quickstart to appear under a certain category, their quickstart must have certain keywords attached to it.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -131,6 +131,11 @@ authors:
   - Author One
   - Other Author
 
+# These keywords can be mapped to category terms from
+# a datasource, which help to categorize quickstarts.
+# To have your quickstart appear under a certain category,
+# these keywords must include a matching category term,
+# see link in the keywords row of the table below.
```
Comment (josephgregoryii) [inline CONTRIBUTING.md:138]:
```suggestion
# These keywords can be mapped to category terms
# which help to categorize quickstarts.
# To have your quickstart appear under a certain category,
# these keywords must include a matching category term,
# see link in the keywords row of the table below.
```

**Review by josephgregoryii — COMMENTED** [review]
Last small suggestions! Ready after

**Review by josephgregoryii — APPROVED** [review]

**Pull request merged**